### PR TITLE
Functional tests - Add new test 'Check summary' on create order page

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'Check summary' on create order page
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | no
| How to test?      | no need
| Possible impacts? | no

### CODE REVIEW

File: tests/UI/pages/BO/orders/add.js [hunk 1]
```diff
@@ -797,6 +812,36 @@ class AddOrder extends BOBasePage {
...
+    if (action === 'Send pre-filled order to the customer by email') {
+      await this.waitForSelectorAndClick(page, this.sendOrderMailButton);
+
+      return this.getTextContent(page, this.summarySuccessMessageBlock);
+    }
+    if (action === 'Proceed to checkout in the front office') {
+      const newPage = await this.openLinkWithTargetBlank(page, this.proceedOrderLink, 'body a');
+
+      return newPage;
+    }
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/orders/add.js]:
```suggestion
   return this.openLinkWithTargetBlank(page, this.proceedOrderLink, 'body a');
```
if there is only two options ou can do it like

File: tests/UI/pages/BO/orders/add.js [hunk 2]
```diff
@@ -797,6 +812,36 @@ class AddOrder extends BOBasePage {
...
+  /**
+   * Set more actions
+   * @param page {Page} Browser tab
+   * @param action {string} Action to select
+   * @returns {Promise<Page|string>}
+   */
+  // eslint-disable-next-line consistent-return
+  async setMoreActions(page, action) {
+    await this.waitForSelectorAndClick(page, this.moreActionsDropDownButton);
+    if (action === 'Send pre-filled order to the customer by email') {
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/orders/add.js]: if the action is just some value that you add to the function and not some value that you get from the page, it does not need to be this long.

`Send pre-filled order to the customer by email` => `Pre-filled order`
`Proceed to checkout in the front office` => `Proceed to checkout`

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/10_checkSummary.js [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+    });
+
+    it('should check that summary block is not visible', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkSummaryNotVisible', baseContext);
+
+      const isSummaryBlockVisible = await addOrderPage.isSummaryBlockVisible(page);
+      await expect(isSummaryBlockVisible, 'Summary block is visible!').to.be.false;
+    });
+
+    it(`should add to cart '${Products.demo_12.name}'`, async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/10_checkSummary.js:124]:
```suggestion
    it(`should add to cart '${Products.demo_11.name}'`, async function () {
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback (and I approved those of @boubkerbribri).

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/10_checkSummary.js (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/01_orders/createOrders/10_checkSummary.js:124]: Done thx :+1:

File: tests/UI/pages/BO/orders/add.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/add.js]: Done thx :+1:

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**